### PR TITLE
pssolar.rst and solar_common.rst_: more on specifying time and add use of -I+z

### PR DESCRIPTION
**Pull request by anbj**

### CODE REVIEW

**Comment by meghanrjones** [comment]
Could you please make the same change to the example in doc/rst/source/solar.rst as in doc/rst/source/pssolar.rst?

**Comment by anbj** [comment]
Will do as soon as I can.

**Comment by anbj** [comment]
Done.

File: doc/rst/source/solar.rst [hunk 1]
```diff
@@ -40,9 +40,9 @@ Examples
 
 .. include:: explain_example.rst_
 
-Print current Sun position and Sunrise, Sunset times at::
+Print current Sun position and Sunrise, Sunset times at given date, time and time zone::
 
-    gmt solar -I-7.93/37.079+d2016-02-04T10:01:00
+    gmt pssolar -I-7.93/37.079+d2016-02-04T10:01:00+z02:00
```
Comment (meghanrjones) [inline doc/rst/source/solar.rst:45]:
```suggestion
    gmt solar -I-7.93/37.079+d2016-02-04T10:01:00+z02:00
```

**Review by meghanrjones — APPROVED** [review]
Looks good, thanks @anbj!

**Comment by anbj** [comment]
My pleasure!

**Pull request merged**